### DISCUSSION TITLE
Showcase

### DISCUSSION THREAD

**Discussion by realistiqrp** [matsn0w/IIS-EmergencyLS-ELS-FiveM#62] [Q&A] [answered]
Is er soms een showcase beschikbaar, zowel van het script als van de ui voor de els te bedienen?

**Comment by matsn0w** [comment, marked as answer]
Ik heb op dit moment nog geen video gemaakt. Wel een goed idee, wellicht dat die nog komt. Je kunt wel de video van de oorspronkelijke versie van deze resource bekijken: https://www.youtube.com/watch?v=MZnO9eIjFWA
Er is (nog!) geen UI voor het bedienen van ELS.

---

I have not made any video at this time. Good idea, maybe it will come. You can watch the video of the original version of this resource: https://www.youtube.com/watch?v=MZnO9eIjFWA
There is no UI (yet!) for operating ELS.
